src/main/java/com/mindera/school/spaceshiprent/persistence/entity/UserEntity.java

```diff
@@ -29,10 +14,18 @@
 @AllArgsConstructor
 @Entity
 @Table(name = "users")
-
 public class UserEntity {
+
     @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @GeneratedValue(
```

Comment:
why did you prefer to use the SEQUENCE strategy?

Reply:
this was a suggestion made by Marcos. He told us we wouldn't depend on jpa, everything would be exactly as we wanted since we implemented it in our own way.